infra/modules/azure_monorepo_single_env_starter_pack/github_environments_cd.tf

```diff
@@ -0,0 +1,100 @@
+resource "github_repository_environment" "infra_prod_cd" {
+  environment = "infra-prod-cd"
+  repository  = github_repository.this.name
+
+  deployment_branch_policy {
+    protected_branches     = false
+    custom_branch_policies = true
+  }
```

Comment:
why not 

```
deployment_branch_policy {
    protected_branches     = true  # Changed to true
    custom_branch_policies = false # Changed to false
  }
```

?

Reply:
I don't think you can set both on true, but I agree to enable `protected_branches` on CD envs